node-feature-discovery-job.json.template

```diff
@@ -37,10 +37,24 @@
                 "containerPort": 7156,
                 "hostPort": 7156
               }
+            ],
+            "volumeMounts": [
+              {
+                "name": "sys",
+                "mountPath": "/sys"
```

Comment:
Mount it in `/host-sys` to differentiate between sys fs inside container and host sys fs?